src/Symfony/Component/Console/Attribute/Option.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Attribute;
+
+use Symfony\Component\Console\Completion\CompletionInput;
+use Symfony\Component\Console\Completion\Suggestion;
+use Symfony\Component\Console\Exception\LogicException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+
+#[\Attribute(\Attribute::TARGET_PARAMETER)]
+class Option
+{
+    private ?int $mode = null;
+    private string $typeName = '';
```

Comment:
if you don't want to add the static property as suggested bellow, it might be worth documenting it via a `@var` attribute? Otherwise you need to do some digging in order to find what this value corresponds to

Reply:
thank you for the suggestion! I believe adding a @var attribute isn't necessary in this case, as the property's purpose and name is already clear within the context